.github/workflows/verify_docker_image.yml

```diff
@@ -1,18 +1,22 @@
 name: Pull and Verify Docker Image
 
 on:
-  workflow_call:
   workflow_dispatch:
+  workflow_call:
+    inputs:
+      amdgpu_target:
+        required: true
+        type: string
 jobs:
   pull-and-build-image:
-    name: Verify Docker Image
+    name: Verify  ${{ inputs.amdgpu_target }} Docker Image
     runs-on: linux-mi300-1gpu-ossci-rocm
     steps:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - name: Pull Docker Image
         run: |
-          docker pull ghcr.io/rocm/therock_pytorch_dev_ubuntu_24_04_gfx942:main
+          docker pull ghcr.io/rocm/therock_pytorch_dev_ubuntu_24_04_${{ inputs.amdgpu_target }}:main
```

Comment:
Either
* the version to verify should be an input that is set on workflow_call **and** workflow_dispatch
* the workflow could be merged into the publish_pytorch_dev_docker one as there right now is no real benefit of having this separated